From 3a78c638891575aa11000686758ce930a6066e22 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 30 Apr 2020 22:32:05 +0300
Subject: [PATCH] [SW-0] goya2_arc: fix comment

uboot_start() function loads U-Boot, not Preboot - fix the comment.

Change-Id: I86e49afd5712789d4f0a67cec38dfc175ab26fbf
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/goya2_arc_gnu/common.c b/goya2_arc_gnu/common.c
index 1fe97be..2b74037 100644
--- a/goya2_arc_gnu/common.c
+++ b/goya2_arc_gnu/common.c
@@ -226,7 +226,7 @@
 void uboot_start(void)
 {
 	unsigned int lsb = 0xf0000000 | UBOOT_TEXT_BASE, msb = 0x7f;
-	/* Set CPU0 Reset address to PREBOOT */
+	/* Set CPU0 Reset address to U-Boot */
 	regmap_writel(lsb, mmCPU_CA53_CFG_RST_ADDR_LSB_0);
 	regmap_writel(msb, mmCPU_CA53_CFG_RST_ADDR_MSB_0);
 
